src/property.h

```diff
@@ -89,10 +89,11 @@ struct Property : public detail::Property {
     PROPERTY(JAHR, 0x0124, Type::et_little_endian);
     PROPERTY(STUNDE, 0x0125, Type::et_little_endian);
     PROPERTY(MINUTE, 0x0126, Type::et_little_endian);
-    PROPERTY(HYSTERESE_WW, 0x0140,et_dec_val);
+    PROPERTY(HYSTERESE_WW, 0x0140, Type::et_dec_val);
     PROPERTY(BETRIEBS_STATUS, 0x0176);
     PROPERTY(VOLUMENSTROM, 0x01da, Type::et_dec_val);
     PROPERTY(TAUPUNKT_HK1, 0x0264, Type::et_dec_val);
+    PROPERTY(HEIZGASTEMPERATUR, 0x0265, Type::et_dec_val); 
```

Comment:
I thought your code was intended for the THZ_504. Therefore, all values should belong to the base, and only deviations need to be highlighted. Otherwise, we have a lot of work because many values work for both the 5.5ECO and the 504, and we would have to duplicate everything.

Reply:
I use VSCode on HA and also VSCode on the desktop, but I have two main problems (besides many others ;-)):

- I use Azure to privately hold the repository of my full HA, and I don’t know how to use GitHub as a second repository to only publish the THZ504 part, which I can share.

- Since you didn’t incorporate much of what I sent over, my code now differs significantly from yours, and Git wants to send all changes at once. However, since you want everything sorted into separate PRs by topic, I have to manually copy from left to right and be careful not to forget half of it. For example, I completely restructured your energy sensors and in doing so, I also found the error with the incorrect values. I couldn’t upload it to GitHub yet because it would otherwise be included in the open PR.

Reply:
> I thought your code was intended for the THZ_504. Therefore, all values should belong to the base, and only deviations need to be highlighted. Otherwise, we have a lot of work because many values work for both the 5.5ECO and the 504, and we would have to duplicate everything.

In the beginning yes, but now with the variant handling it makes sense to have different configs. The duplication part is very small and will be gone once it is tested with all variants. Also essentially it is copy paste and nothing to do from scratch